### PR TITLE
*: bump to 2.0.5

### PR DESCRIPTION
**Pull request by BusyJay**

### CODE REVIEW

**Comment by disksing** [comment]
LGTM

**Comment by BusyJay** [comment]
/run-integration-tests pd=release-2.0 tidb=release-2.0 tidb-tests=release-2.0

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 All notable changes to this project are documented in this file.
 See also [TiDB Changelog](https://github.com/pingcap/tidb/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [2.0.5]
+### Bug fixes
+* Fix the potential overflow issue in decimal operations
+* Fix the dirty read issue that might occur in the process of merg
```
Comment (overvenus) [inline CHANGELOG.md]: typo, `merge`.

**Comment by BusyJay** [comment]
PTAL

**Comment by BusyJay** [comment]
/run-integration-tests pd=release-2.0 tidb=release-2.0 tidb-test=release-2.0

**Pull request merged**